deployment/terraform/api.tf

```diff
@@ -0,0 +1,143 @@
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  name   = "sg${var.project}APILoadBalancer"
+  vpc_id = data.terraform_remote_state.core.outputs.vpc_id
```

Comment:
Due to the need to support multiple environment of different types, I believe that we need to make this a Terraform variable and then populate its value differently based on whether we're targeting our own AWS accounts, or the AWS accounts of others.